go-controller/cmd/ovnkube-trace/ovnkube-trace.go

```diff
@@ -501,14 +523,27 @@ func getPodInfo(coreclient *corev1client.CoreV1Client, restconfig *rest.Config,
 		podInfo.OfportNum = ovsInterfaceInformation.Ofport
 	}
 
-	podInfo.NodeExternalBridgeName, err = getNodeExternalBridgeName(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, cmd, podInfo.NodeName)
+	podInfo.NodeExternalBridgeName, err = getNodeExternalBridgeName(coreclient, restconfig, ovnNamespace, podInfo.OvnKubePodName, podInfo.SbCommand, podInfo.NodeName)
 	if err != nil {
 		return nil, err
 	}
 
 	return podInfo, err
 }
 
+func getSwitchPortMacAddress(coreclient *corev1client.CoreV1Client, restconfig *rest.Config, podInfo *PodInfo, ovnNamespace, portPrefix string) (string, error) {
```

Comment:
nit: leave doc strings and mention this is only used for IC=true?

Reply:
This method is used for retrieving both rtos and rtots mac, The portPrefix is the differenciator here.